src/utils/utils.js

```diff
@@ -297,6 +297,32 @@ const pickFork = (...forks) => props => {
  */
 const isEthStrictAddress = address => /^0x[0-9a-f]{40}$/i.test(address)
 
+const getTimeRangeByDuration = timeWindow => {
+  const unit = timeWindow.slice(-1)
+  const timeAmount = timeWindow.slice(0, -1)
+  return {
+    to: moment()
+      .startOf('hour')
+      .toISOString(),
+    from: moment()
+      .startOf('hour')
+      .subtract(timeAmount, unit)
+      .toISOString(),
+    timeWindow
+  }
+}
+
+const mapItemsToKeys = (items, { keyPath, getKeyPath }) => {
+  const obj = {}
+
+  items.forEach(item => {
```

Comment:
use reduce if you want to return one variable obj. @devanshu-santiment 
Also, it fixed mutation of const obj variable. Sure it's JS and it's ok, but better to keep the rule if const than it's const.